includes/polling/os/gw-eydfa.inc.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * gw-eydfa.inc.php
+ *
+ * LibreNMS OS poller module for Glass Way WDM EYDFA Optical Amplifier
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2018 TheGreatDoc
+ * @author     TheGreatDoc
+ */
+
+$hardware = snmp_get($device, ".1.3.6.1.4.1.17409.1.3.3.2.2.1.4.1", "-OQv");
```

Comment:
Do you not have the OID names to use here? Also needs converting to multi gets.

Reply:
Yes. I dont know why I used numeric.